python/cuml/common/cuda.pyx

```diff
@@ -16,6 +16,14 @@
 
 # distutils: language = c++
 
+
+from rmm._cuda.gpu import getDeviceCount
+
+
+def is_cuda_available():
+    return getDeviceCount() >= 1
```

Comment:
This should be fine, but we might want to consider a more robust approach at some point